app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
```

Comment:
Pycharm does recommendation about this, however it is not a requirement to mark this class as an ABC. In terms of functionality it will behave the same with or without